prototypes/anim_polygons_data/shared/render.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
+import os
+import argparse
+
+
+def run_blender(scene, out_dir):
+    os.system("xvfb-run --auto-servernum "
+              "--server-args='-screen 0 1600x1024x16' "
+              f"blender --background {scene} --render-output "
+              f"{out_dir}/frame####.png --render-anim -noaudio")
+
+
+def clean_dir(out_dir):
```

Comment:
Why not use [shutil.rmtree](https://docs.python.org/3/library/shutil.html#shutil.rmtree) ?